src/Microsoft.SqlTools.ServiceLayer/QueryExecution/ResultSet.cs

```diff
@@ -791,8 +791,8 @@ private async Task<long> AppendRowToBuffer(DbDataReader dbDataReader)
             {
                 throw new InvalidOperationException(SR.QueryServiceResultSetAddNoRows);
             }
-            
-            using (IFileStreamWriter writer = fileStreamFactory.GetWriter(outputFileName))
+
+            using (IFileStreamWriter writer = fileStreamFactory.GetWriter(outputFileName, null))
```

Comment:
Could specify null as the default argument, so you don't have to pass it in explicitly.

Reply:
Ok, change made. Though I still intend to do the refactor from the abandoned PR that will make the unused columns parameter go away.